tests/playbooks/tests_ethtool_features.yml

```diff
@@ -132,28 +136,34 @@
                 TEST: Change feature with both underscores and dashes.
               debug:
                 msg: "##################################################"
-            - network_connections:
-                provider: "{{ network_provider | mandatory }}"
-                connections:
-                  - name: "{{ interface }}"
-                    state: up
-                    type: ethernet
-                    ip:
-                      dhcp4: "no"
-                      auto6: "no"
-                    ethtool:
-                      features:
-                        tx_tcp_segmentation: "no"
-                        tx-tcp-segmentation: "no"
-                __header: "# Ansible managed test header"
-              register: __network_connections_result
+            - name: Import network role
+              import_role:
+                name: linux-system-roles.network
+              vars:
+                network_connections:
+                  provider: "{{ network_provider | mandatory }}"
+                  connections:
+                    - name: "{{ interface }}"
+                      state: up
+                      type: ethernet
+                      ip:
+                        dhcp4: "no"
+                        auto6: "no"
+                      ethtool:
+                        features:
+                          tx_tcp_segmentation: "no"
+                          tx-tcp-segmentation: "no"
+                  __header: "# Ansible managed test header"
```

Comment:
```suggestion
```
The reason this was here was that the old code was calling the module directly, which needed to add the `__header` - since you changed it to use the role, you should omit `__header`, since the role will add it